src/profile.ts

```diff
@@ -346,40 +414,63 @@ export default class Profile {
     this.filamentDensity = new Array(extruderCount).fill(1.25);
     this.filamentDeretractSpeed = new Array(extruderCount).fill('nil');
     this.filamentDiameter = new Array(extruderCount).fill(1.75);
-    this.filamentLoadTime = new Array(extruderCount).fill(0);
```

Comment:
I think we should keep the class properties sorted the same as the output order in `toString()` below; I know they're not sorted alphabetically here but I think that consistency is more important